frontend-instructor/src/components/navBar/NavItems.tsx

```
@@ -0,0 +1,55 @@
+<<<<<<< HEAD:frontend-instructor/src/components/navBar/NavItems.jsx
+import { Link, useLocation } from "react-router-dom";
+=======
+import useInstructorAuth from "@//context/authContext";
+import { Link } from "react-router-dom";
+>>>>>>> b649237cb593cac19930a6a6e1ea328feea57ae4:frontend-instructor/src/components/navBar/NavItems.tsx
+import { toast, Toaster } from "sonner";
+import useInstructorAuth from "@/src/context/authContext";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'useInstructorAuth'. Consider to delete it or rename it.

'useInstructorAuth' is defined here:


(lint/suspicious/noRedeclare)